AK/UFixedBigInt.h

```diff
@@ -16,481 +18,414 @@
 #include <AK/StringBuilder.h>
 
 namespace AK {
+namespace Detail {
+// As noted near the declaration of StaticStorage, bit_size is more like a hint for a storage size.
+// The effective bit size is `sizeof(StaticStorage<...>) * 8`. It is a programmer's responsibility
+// to ensure that the hinted bit_size is always greater than the actual integer size.
+// That said, do not use unaligned (bit_size % 64 != 0) `UFixedBigInt`s if you do not know what you
+// are doing.
+template<size_t bit_size, typename Storage = Detail::StaticStorage<false, bit_size>>
```

Comment:
if we reference StaticStorage via Detail here, we don't need to export it in the previous commit.